cernopendata/modules/fixtures/data/docs/delphi-news-first-release/delphi-news-first-release.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "author": "DELPHI Collaboration",
+    "body": {
+      "content": "delphi-news-first-release.md",
+      "format": "md"
+    },
+    "date_published": "2024-06-26",
+    "experiment": [
+      "DELPHI"
+    ],
+    "short_description": {
+      "content": "The DELPHI Collaboration has released the first set of its data samples for the Open Data Portal. Regardless if you are a researcher, teacher, student or just interested, start your journey through this exciting first dataset from LEP."
```

Comment:
"first set of its data samples" -> will there be more? If not, we should rephrase.